akka-http-tests/src/test/java/akka/http/javadsl/server/directives/ParameterDirectivesTest.java

```diff
@@ -34,8 +34,8 @@ public void testStringParameterExtraction() {
 
     route
       .run(HttpRequest.create().withUri("/abc?stringParam=a=b"))
-      .assertStatusCode(400)
```

Comment:
Keep the old assertions but use a different faulty query, we want to test the faulty case here.